### PR TITLE
Des 129 display courses topics path dynamic

### PR DESCRIPTION
**Pull request by cristianarce89**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
 REACT_APP_API_URL=https://regresemos-cms.herokuapp.com/api
-# REACT_APP_API_URL=https://localhost:8080/api
+# REACT_APP_API_URL=http://localhost:8080/api
```
Comment (DanielCalleCO) [inline .env:2]: Why did this URL change?

File: src/database/videos.js [hunk 2]
```diff
@@ -1,12 +1,18 @@
 import { useAuthStore } from '../hooks/useAuthStore';
 
-const videos = async () => {
+export const videos = async () => {
   const { videosLearningPath } = useAuthStore;
   const videosrender = videosLearningPath.map((videos) => {
```
Comment (DanielCalleCO) [inline src/database/videos.js:5]: Could you please replace `videosrender` by `videosRender` please?

```suggestion
  const videosRender = videosLearningPath.map((videos) => {
```

File: src/database/videos.js [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+    return {
+      key: key,
+      title: title,
+      url: url,
+    };
   });
-};
 
-videos();
+  return videosrender;
```
Comment (DanielCalleCO) [inline src/database/videos.js:17]:
```suggestion
  return videosRender;
```

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]
Your code is clean and easy to understand, thanks! Please review a couple of suggestions I have.

**Review by DanielCalleCO — APPROVED** [review]
GOOD JOB! LGTM

**Review by Daniielmayo — APPROVED** [review]

**Pull request merged**